modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Filters.tsx

```diff
@@ -141,9 +150,24 @@ function AddFDSFilterModalContent({
 
 		let body: any = {
 			fieldName: selectedField.name,
-			name: name || selectedField.label,
 		};
 
+		if (Liferay.FeatureFlags['LPS-172017']) {
+			let labels;
+			if (!translationExists({translations: i18nFilterLabels})) {
+				const defaultLanguageId = Liferay.ThemeDisplay.getDefaultLanguageId();
+				labels = {[defaultLanguageId]: selectedField.label};
```

Comment:
Save fieldName as the default language in the same way we do without translations. We need the value to show in the filters table.
![Screenshot from 2023-09-26 08-49-40](https://github.com/liferay-frontend/liferay-portal/assets/132653342/6c51f9e4-349c-48fb-a882-88dcd149bb42)

Another option is waiting for the filter list and modify the response if no label is available

Reply:
This is preferrable from a UI perspective. Considering that user can change labels at will, I don't see any reason not to implement this